### DISCUSSION TITLE
NFT-DAO Metadata github use and development

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/NFT-Metadata#3] [Ideas]
Hi there Michael, 

I thought I would post notes here to be transparent. 

I have written a draft landing page for NFT-Metadata on my github at https://github.com/stephen-rowan/Cardano-NFT-DAO-Notes/tree/main/Metadata (I thought it would be easier to do this rather than fork from here at this early stage). 

This includes a project README.md summary (with notes) and possible folder layout. Also I drafted a Meetings record at https://github.com/stephen-rowan/Cardano-NFT-DAO-Notes/blob/main/Metadata/Meetings/2021-03-09-Metadata-github.md (based on our discussions yesterday). 

Feel free to cut and paste anything you want into this github. Speak to you later.

All the best

Stephen

**Comment by Yagi-Michael** [comment]
This is a great start! Thank you for doing this :)

**Reply by stephen-rowan** [reply]
You are welcome Michael 👍 

I have now moved over all the markdown content we discussed yesterday (as is). Only added a reference to the MIT Open Source License which may be superfluous in this context. All amendments under "1.0 - Move over draft markdown layout". I have briefly checked all the inline links to check they point here. I will now begin to retire the "Cardano-NFT-DAO-Notes" repository on my github in order to work here.

**Comment by kopcho** [comment]
We should be using the same FOSS license as IOHK which I believe is Apache.
________________________________

**Reply by stephen-rowan** [reply]
OK good to know 👍 .

 I have changed the reference to the Apache-2.0 license (https://opensource.org/licenses/Apache-2.0).
